en/docs/learn/configuring-identity-analytics.md

```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
 
 A taxi company called "Pickup" has launched a new application to be used by their customers. The Pickup developers wish to measure the performance of the authentication mechanism used for users to log into the application so that they can improve the user login experience. 
 
-To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application. 
+To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application.
 
 ## Set up 
 
 1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
 
-2. [Download WSO2 Identity Server Analytics distribution](https://wso2.com/identity-and-access-management/install/analytics/).
+2. [Download WSO2 Identity Server Analytics distribution](https://github.com/wso2/analytics-is/releases/tag/v5.8.0-rc3).
```

Comment:
Why are we downloading an RC pack here?

Reply:
For the 5.11.0 branch the given url is this. But for the master branch as you can see the above url is invalid. Therefore I have override the provided url with the 5.11.0 branch given url.